resources/views/layout.blade.php

```diff
@@ -0,0 +1,96 @@
+<!DOCTYPE HTML>
+<head>
+    <title>Store Website</title>
+    <meta http-equiv  =  "Content-Type" content  =  "text/html; charset = utf-8" />
+    <meta name = "viewport" content  =  "width = device-width, initial-scale = 1, maximum-scale = 1">
+    <link href = "{{ asset('css/style.css') }}" rel = "stylesheet" type = "text/css" media = "all" />
+    <link href = "{{ asset('css/style1.css') }}" rel = "stylesheet" type = "text/css" media = "all" />
+    <link href = "{{ asset('css/bootstrap.css') }}" rel = "stylesheet" type = "text/css" media = "all" />
+    <link href="{{ asset('css/comment.css') }}" rel="stylesheet" type="text/css" media="all" />
+    <link href = "{{ asset('css/menu.css') }}" rel = "stylesheet" type = "text/css" media = "all" />
```

Comment:
```suggestion
    <link href="{{ asset('css/menu.css') }}" rel = "stylesheet" type="text/css" media="all" />
```
fix tương tự các dòng khác nhá